addon/components/yeti-table/tbody/row/cell.hbs

```diff
@@ -1,4 +1,4 @@
-{{#if this.visible}}
+{{#if this.column.visible}}
```

Comment:
So it seems like a change here, as the cell itself can't have its own visible flag anymore.
This is probably fine (I don't see why a column would be visible, but some of the cells wouldn't), but that's a breaking change to me.

Reply:
Technically the visible flag is passed from the column, although the user could override it, it leads to problematic HTML, which is why I didnt write it in such a way as to provide the ability to override it. I dont feel it should be documented as a user defined field for just this reason. Just because it was available, doesnt mean you should :) 

I believe this is going to be a major bump anyway.

Reply:
100% agree :)